src/set_theory/cardinal.lean

```diff
@@ -938,7 +942,7 @@ end
 
 @[simp] lemma omega_add_omega : ω + ω = ω := mk_denumerable _
 
-@[simp] lemma omega_mul_omega : ω * ω = ω := mk_denumerable _
+lemma omega_mul_omega : ω * ω = ω := mk_denumerable _
```

Comment:
how come the simp attribute is still needed on `omega_add_omega` but not on `omega_mul_omega`? Is there a missing simp lemma somewhere, that would say that if alpha is infinite then `alpha + omega = alpha`? (the analogue of `omega_mul_eq`)

Reply:
I'll add the missing `@[simp]` attribute in another PR.